dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -44,7 +44,7 @@ def login
     # from the LTI Platform, and can use it to query for these values in the
     # authenticate controller action.
     begin
-      write_cache(state_and_nonce[:state], state_and_nonce)
+      write_cache(state_and_nonce[:state], state_and_nonce, 15.minutes)
```

Comment:
I could have updated the default expiration, however I chose to explicitly override it here. I could go either way, this just felt a little more clear.